src/theory/sets/theory_sets_private.cpp

```diff
@@ -332,17 +337,23 @@ void TheorySetsPrivate::fullEffortCheck()
             tnct = n;
           }
         }
-        // register it with the state
+        // register it with the term registry
```

Comment:
`d_state.registerTerm(eqc, tnn, n);` below does not register any node with the term registry. 
The `StateSovler` has no knowledge of `TermRegistry`.

Reply:
This is a typo, I'll revert this update to the comment.